router/smoke/openshift_2/conftest.py

```diff
@@ -92,7 +78,9 @@ def receiver(request, iqa) -> Union[ReceiverJava, ReceiverPython, ReceiverNodeJS
         s: str = request.param.split('_')
         receiver_implementation = s[1]
         receiver_number = int(s[2])
-        return iqa.get_clients(Receiver, receiver_implementation)[receiver_number]
+        receiver = iqa.get_clients(Receiver, receiver_implementation)[0]
+        receiver.set_url("amqp://%s:5672/address" % request.config.option.cluster[receiver_number])
+        return receiver
```

Comment:
Since we are no longer defining the URL through the inventory file, due to the dynamic nature of having cross senders x receivers x routers, I am setting it while defining the fixture.